conformance/src/main/java/build/buf/Main.java

```diff
@@ -88,13 +89,12 @@ static TestResult testCase(
   private static TestResult validate(Validator validator, DynamicMessage dynamicMessage) {
     try {
       ValidationResult result = validator.validate(dynamicMessage);
-      if (result.getViolations().isEmpty()) {
+      List<Violation> violations = result.getViolations();
+      if (violations.isEmpty()) {
         return TestResult.newBuilder().setSuccess(true).build();
       } else {
-        return TestResult.newBuilder()
-            .setValidationError(
-                Violations.newBuilder().addAllViolations(result.getViolations()).build())
-            .build();
+        Violations error = Violations.newBuilder().addAllViolations(violations).build();
```

Comment:
Small style thing (not from this change) but typically we shouldn't have an `else` after a `return` in the previous if block.